frontend/src/metabase/core/components/AutocompleteInput/AutocompleteInput.stories.tsx

```diff
@@ -30,3 +31,44 @@ const Template: ComponentStory<typeof AutocompleteInput> = args => {
 };
 
 export const Default = Template.bind({});
+
+export const CustomFilter = Template.bind({});
+CustomFilter.args = {
+  filterFn: (value: string | undefined, options: string[]) => {
+    if (!value) {
+      return [];
+    } else {
+      return options.filter(o => o.includes(value[0]));
+    }
+  },
+};
+
+const CustomOptionClickTemplate: ComponentStory<
+  typeof AutocompleteInput
+> = args => {
+  const [value, setValue] = useState("");
+
+  const optionSelectHandler = (option: string) => {
```

Comment:
nit: we usually name event handlers like this `handleXXX`, e.g. `handleOptionClick` or something.